src/index.tsx

```diff
@@ -46,6 +47,33 @@ type EventLog = {
   timestamp: Date;
 };
 
+type verifyServerExtensiontProps = {
+  api: SchedulerService;
+  trans: IRenderMime.TranslationBundle;
+};
+
+/**
+ * Call API to verify that the server extension is actually installed.
+ */
+async function verifyServerExtension(props: verifyServerExtensiontProps) {
```

Comment:
Using an inline type definition is preferable for simple, local types (i.e. ones that are not exported):

```suggestion
async function verifyServerExtension(props: {
  api: SchedulerService;
  trans: IRenderMime.TranslationBundle;
}) {
```

This syntax is significantly shorter and allows developers to easily see the type of `props` without having to look up a type definition above.

I'm leaving this up to you. However, if you want to keep the named type, you should correct the spelling and capitalize it per convention.